### PR TITLE
N26: Learn WGPU section

### PR DESCRIPTION
**Pull request by sotrh**
_Part of #783_

### CODE REVIEW

**Comment by AngelOnFira** [comment]
@sotrh Any idea on why there are so many old commits as part of this branch? It might be caused by your source not being up to date with the origin's. We do squash changes though, so it shouldn't be an issue. Might be worth rebasing though.

File: content/news/026/index.md [hunk 1]
```diff
@@ -102,6 +102,17 @@ as [several key differences remain].
...
+### Learn WGPU Updated: No more swap chains!
+
+As part of the update to 0.10 the wgpu team removed the `SwapChain` from the
+API. The `surface` will now be used to retrieve textures to render to wrapped
+in `SurfaceTexture`s. You configure the `Surface` in a similar way to how you
+would configure the `SwapChain` except the struct is now called
+`SurfaceConfiguration` instead of `SwapChainDescriptor`. If you want to know
+more, you can check [the tutorial].
+
+[the tutorial]: https://sotrh.github.io/learn-wgpu/news
```
Comment (AngelOnFira) [inline content/news/026/index.md:114]: Can you link to the permalink instead?

```suggestion
[the tutorial]: https://sotrh.github.io/learn-wgpu/news/#_0-10-swapchain-is-dead-long-live-the-surface
```

File: content/news/026/index.md [hunk 2]
```diff
@@ -102,6 +102,17 @@ as [several key differences remain].
 
 ## Learning Material Updates
 
+### Learn WGPU Updated: No more swap chains!
+
+As part of the update to 0.10 the wgpu team removed the `SwapChain` from the
```
Comment (AngelOnFira) [inline content/news/026/index.md:107]:
```suggestion
As part of the update to 0.10 the wgpu team removed `SwapChain` from the
```

**Review by AngelOnFira — CHANGES_REQUESTED** [review]
Some small improvements.

**Comment by sotrh** [comment]
> @sotrh Any idea on why there are so many old commits as part of this branch? It might be caused by your source not being up to date with the origin's. We do squash changes though, so it shouldn't be an issue. Might be worth rebasing though.

Yeah I should probably rebase.

**Pull request closed without merging**